spec/dummy/.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.6.3
```

Comment:
Not sure what these `dummy` files are for, but this version is different from the `.ruby-version` one at root

Reply:
`dummy` is just a mock application to test rails plugins on.
https://medium.com/@sarahbranon/testing-rails-engines-the-dummy-app-d20d25c20466